ocs_ci/utility/ibmcloud.py

```diff
@@ -1242,6 +1243,50 @@ def find_free_network_subnets(subnet_cidr, network_prefix=27):
             return possible_subnets
 
 
+def delete_dns_records(cluster_name):
+    """
+    Delete DNS records leftover from cluster destroy.
+
+    Args:
+        cluster_name (str): Name of the cluster, used to filter DNS records
+
+    """
+    dns_domain_id = config.ENV_DATA["base_domain_id"]
+    cis_instance_name = config.ENV_DATA["cis_instance_name"]
+    ids_to_delete = []
+    page = 1
+
+    logger.info(f"Setting cis instance to {cis_instance_name}")
+    run_ibmcloud_cmd(f"ibmcloud cis instance-set {cis_instance_name}")
+
+    while True:
+        out = run_ibmcloud_cmd(
+            f"ibmcloud cis dns-records {dns_domain_id} --per-page 1000 --page {page} --output json"
+        )
+        records = json.loads(out)
+        if not records:
+            logger.info("Reached end of pagination")
+            break
+
+        filter_string = f".{cluster_name}."
+        logger.info(f"Searching for records with string: {filter_string}")
+        for record in records:
+            if filter_string in record["name"]:
+                logger.info(f"Found {record['name']}, marking for deletion")
+                ids_to_delete.append(record["id"])
+        page += 1
+
+    logger.info(f"Records to delete: {ids_to_delete}")
+    for record_id in ids_to_delete:
+        logger.info(f"Deleting DNS record: {record_id}")
+        try:
+            run_ibmcloud_cmd(
+                f"ibmcloud cis dns-record-delete {dns_domain_id} {record_id}"
+            )
+        except CommandFailed:
+            logger.error(traceback.format_exc())
```

Comment:
```suggestion
            logger.exception("Failed to delete CIS leftovers")
```

I think that logger.exception give us also details about traceback without need of import and use traceback:

```
In [7]: try:
   ...:     1/0
   ...: except Exception:
   ...:     logger.exception("This is the zero division error")
   ...:
This is the zero division error
Traceback (most recent call last):
  File "<ipython-input-7-8c23c22edfe2>", line 2, in <module>
    1/0
    ~^~
ZeroDivisionError: division by zero
```

Reply:
Removed the import and went with your suggestion